packages/rancher-prom2teams/generated-changes/patch/templates/deployment.yaml.patch

```diff
@@ -0,0 +1,76 @@
+--- charts-original/templates/deployment.yaml
++++ charts/templates/deployment.yaml
+@@ -2,8 +2,8 @@
+ kind: Deployment
+ metadata:
+   name: {{ include "prom2teams.fullname" . }}
+-  labels:
+-{{ include "prom2teams.labels" . | indent 4 }}
++  namespace: {{ include "prom2teams.namespace" . }}
++  labels: {{ include "prom2teams.labels" . | nindent 4 }}
+ spec:
+   replicas: {{ .Values.replicaCount }}
+   selector:
+@@ -17,16 +17,15 @@
+         app.kubernetes.io/instance: {{ .Release.Name }}
+     spec:
+     {{- with .Values.imagePullSecrets }}
+-      imagePullSecrets:
+-        {{- toYaml . | nindent 8 }}
++      imagePullSecrets: {{ toYaml . | nindent 8 }}
+     {{- end }}
+       volumes:
+         - name: config
+           configMap:
+-            name: prom2teams-config
++            name: {{ include "prom2teams.fullname" . }}
+       containers:
+         - name: {{ .Chart.Name }}
+-          image: "{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}"
++          image: {{ include "system_default_registry" . }}{{ .Values.image.repository }}:{{ default .Chart.AppVersion .Values.image.tag }}
```

Comment:
Let's not default to `.Chart.AppVersion` since that can't be assumed for the way we handle private registry today (e.g. a `.Values.image.tag` must exist)

Reply:
fixed.